account_bank_statement_import_online_paypal/models/online_bank_statement_provider_paypal.py

```diff
@@ -277,6 +277,27 @@ def _obtain_statement_data(self, date_since, date_until):
             'balance_end_real': balance_end,
         }
 
+    @api.multi
+    def _get_statement_date_step(self):
+        delta = super()._get_statement_date_step()
+        if self.service == 'paypal':
```

Comment:
https://github.com/OCA/bank-statement-import/blob/12.0/account_bank_statement_import_online/models/online_bank_statement_provider.py#L341 not sure there's any reason to do this

Reply:
I try to be minimally invasive here - if those functions work well for other providers, we shouldn't change the logic for them

Reply:
Probably I've expressed myself poorly: I mean that this specific change to the method shouldn't have any effect since step is either day, week or month - all having no hours minutes etc.

Reply:
when you handle datetimes (as opposed to dates), then time is very relevant. I'll check out what you write below later

Reply:
yes, that's obvious, yet still - it seems to me that this code part effectively changes nothing.